### ISSUE TITLE
Checkbox is not visible to Create shipping label

### ISSUE THREAD

**Issue by dev-ajaysaini** [magento/magento2#29816]
While creating shipment there is no checkbox available in Magento 2.4.0

### Preconditions (*)

1. Magento 2.4.0
2. Place order with a shipping method which have functionality to create shipping label.

### Steps to reproduce (*)

1. Place an order with simple product
2. Choose an shipping method which have feature to create shipment label.
3. Try to create shipping label

### Expected result (*)

Checkbox should be visible to select this option.
https://prnt.sc/u8xybu

### Actual result (*)

https://prnt.sc/u8xxy7

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [x] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [x] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by mam08ixo** [comment]
Relevant CSS:

```css
.admin__control-checkbox + label::before
```

M2.3 markup:

```html
<input id="create_shipping_label" class="admin__control-checkbox" onclick="toggleCreateLabelCheckbox();">
<label for="create_shipping_label"></label>
```

M2.4 markup:

```html
<input id="create_shipping_label" class="admin__control-checkbox">
<script type="text/javascript"></script>
<label for="create_shipping_label"></label>
```

Someone moved the `onclick` JS to a `<script>` element so that the CSS selector no longer matches.

@webkul-ajaysaini You can still open the packaging popup by clicking the "Create Shipping Label" label text. Can you confirm that the packaging popup itself is also completely broken?

![packaging_popup](https://user-images.githubusercontent.com/1771622/93574351-9b0eb900-f998-11ea-9fb2-b876c24dd1d3.png)

**Comment by mam08ixo** [comment]
Looks like MC-29420 (70ede03) broke it and MC-35514 (3c22eef) attempts to fix it.
No shipping on Magento 2.4.0, got to wait for a future release.

**Comment by engcom-Bravo** [comment]
Hello @webkul-ajaysaini 

Thank you for your report. 

This issue is already fixed in 2.4.1 and 2.4-develop Magento versions
The Checkbox for Creating Shipping labels is available
![Selection_030](https://user-images.githubusercontent.com/51680745/96873013-0c042d80-147d-11eb-9544-4969a4891031.png)

We are closing this issue. Please feel free to comment, reopen or create a new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines)
